spec/unit/target/build_settings/aggregate_target_settings_spec.rb

```diff
@@ -190,7 +195,7 @@ def pod_target(spec, target_definition)
             end
 
             it 'links the pod targets with the aggregate target' do
-              @xcconfig.to_hash['OTHER_LDFLAGS'].should.include '-l"BananaLib-Pods"'
+              @xcconfig.to_hash['OTHER_LDFLAGS'].should.include '-l"BananaLib-Pods-SampleProject"'
```

Comment:
this is correct since we are now integrating on a real project and a real user target.